nixos/lib/make-channel.nix

```diff
@@ -1,3 +1,7 @@
+/* Build a channel tarball. These contain, in addition to the nixpkgs
+ * expressions themselves, files that indicate the version of nixpkgs
+ * that they represent. This only seems to be used by Travis.
```

Comment:
Not travis but hydra

Reply:
Yes, I'm not sure where I got that idea from…

Reply:
You were probably listening to [Rae & Christian](https://genius.com/Rae-and-christian-travis-lyrics) ;)